tools/.vscode/launch.json

```diff
@@ -9,7 +9,7 @@
             "type": "python",
             "request": "launch",
             "stopOnEntry": true,
-            "pythonPath": "${config:python.pythonPath}",
+            "pythonPath": "${config:python.interpreterPath}",
```

Comment:
Is this change intended? What is the reason it is changed (I don't know what either does)

Reply:
This change was not intended and I also do not know what it does.